enabler/src/de/schildbach/pte/util/HttpClient.java

```diff
@@ -134,6 +143,9 @@ public InputStream getInputStream(final String urlStr, final String postRequest,
 			final URL url = new URL(urlStr);
 			final HttpURLConnection connection = (HttpURLConnection) url.openConnection();
 
+			if (connection instanceof HttpsURLConnection && sslAcceptAllHostnames)
```

Comment:
Because providers use a HTTP URL. Most public transit authorities have not heard about HTTPS yet.